File: Javascript/02.check_palindrome.js
function isPalindrome(num) {
  let reversed = 0;
  let n = num;

  while (n > 0) {
    const last_digit = n % 10;
    reversed = reversed * 10 + last_digit;
    n = Math.floor(n / 10);
  }

  return num === reversed;
}

console.log(isPalindrome(12345));
console.log(isPalindrome(111));

File: Python/02.check_palindrome.py
def is_palindrome(n):
    num = n
    result = 0
    while num > 0:
        last_digit = num % 10
        result = result * 10 + last_digit;
        num = num//10
        
    if(n == result):
        print(f"{n} is a palindrome")
    else:
        print(f'{n} is not a palindrome')

is_palindrome(1111)        
is_palindrome(12345)        

    



    
    
